src/api/app/models/requests_datatable.rb

```diff
@@ -0,0 +1,140 @@
+class RequestsDatatable
+  delegate :sprite_tag, :common_parts, :user_with_realname_and_icon, :fuzzy_time,
+           :project_or_package_link, :params, :link_to, :request_show_path, to: :@view
```

Comment:
These are view helper methods which I use e.g. in ``show_column`` or ``requester_column``